nodejs/FatBeaconPeripheral/README.md

```diff
@@ -0,0 +1,8 @@
+This is a nodejs implementation of a Fatbeacon developed on a Raspberry Pi 3.
+It will advertise a connectable Fatbeacon over Bluetooth Low Energy (BLE). 
+When the central (client) connects to the peripheral (Fatbeacon/server), the
+central will attempt to read the webpageCharacteristic, which serves a HTML.
+There is also the option to have this data be compressed with gzip.
```

Comment:
Could you write usage instructions? (What npm commands do I have to run, what node commands?  What packages do I have to sudo apt-get install?  Are there other system settings I have to turn on?